.gitignore

```diff
@@ -0,0 +1,24 @@
+# See https://help.github.com/articles/ignoring-files/ for more about ignoring files.
+
+# dependencies
+/node_modules
+/front/node_modules
```

Comment:
Are 4 and 5 lines redundant? Can just node_modules help to ignore all directories called node_modules in the current folder and any subfolders?
https://stackoverflow.com/questions/29820791/git-ignore-node-modules-folder-everywhere

Reply:
fixed